.github/workflows/push-helm.yaml

```diff
@@ -0,0 +1,47 @@
+name: Push Chart to OCI
+
+on: push
```

Comment:
We shouldn't be publishing from non-protected branches.  We should only publish from whichever branch is being considered `main`.